packages/graph/README.md

```diff
@@ -0,0 +1,21 @@
+# The HASH Graph API
+
+## Running the database
+
+1.  In order to setup the database, first the database has to be started:
+
+```shell
+docker-compose -f deployment/stack.yml up
+```
+
+1.  (Optional) If starting the database for the first time, the tables has to be generated. From the `migration` directory, run the following command:
+
+```shell
+yarn graph:recreate-db
+```
+
+1.  Then, apply the migration from the `migration` directory:
```

Comment:
```suggestion
1.  Then, apply the migrations from the `migration` directory:
```

Presumable this is going to apply all of them? (even if we have one at the moment) Or will it just apply the latest or something.

Reply:
I guess you are right. @benwerner01 can you confirm?